internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -1351,9 +1354,12 @@ private Map<CharSequence, CharSequence> getDebugInfo(Map<String, String> debugIn
       CharSequence defaultValue = debugInfo.get(k);
       if (defaultValue != null && !defaultValue.equals(new Utf8(v))) {
         logger.warn(
-            "Debug info key '" + k + "' will be omitted because it is already part of the default "
-                + this.getClass().getSimpleName() + " debug info. " + "Default value: '" + defaultValue + "', "
-                + "supplied (omitted) value: '" + v + "'.");
+            "Debug info key: {} will be omitted because it is already part of the default {} debug info. Default value: {}, "
```

Comment:
The key, default value and supplied (ommitted) value should be put between single quotes.

Reply:
Nisarg, thanks for the review and explain the conventions. Will add them back.